tests/ctst/steps/bucket-policies/utils.ts

```diff
@@ -0,0 +1,500 @@
+    'PutObjectVersionLegalHold',
+];
+
+const needVersioning = [
+    'PutBucketReplication',
+    'ListObjectVersions',
+    'DeleteObjectVersion',
+    'DeleteObjectVersionTagging',
+    'GetObjectVersion',
+    'GetObjectVersionAcl',
+    'GetObjectVersionTagging',
+    'PutObjectVersionAcl',
+    'PutObjectVersionTagging',
+    'PutObjectVersionRetention',
+    'PutObjectVersionLegalHold',
+];
+
+const actionPermissions: ActionPermissionsType[] = [
+    // Create bucket is not checking any bucket policy
+    // {
+    //     action: 'CreateBucket',
+    //     permissions: ['s3:CreateBucket'],
+    //     expectedResultOnAllowTest: 'BucketAlreadyOwnedByYou',
+    //     excludePermissionOnBucketObjects: true,
+    //     useWildCardBucketName: true,
+    // },
+    {
+        action: 'PutBucketAcl',
+        permissions: ['s3:PutBucketAcl'],
+        expectedResultOnAllowTest: 'MalformedXML',
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketEncryption',
+        permissions: ['s3:PutEncryptionConfiguration'],
+        expectedResultOnAllowTest: 'MalformedXML',
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketLifecycleConfiguration',
+        permissions: ['s3:PutLifecycleConfiguration'],
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketNotificationConfiguration',
+        permissions: ['s3:PutBucketNotification'],
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketPolicy',
+        permissions: ['s3:PutBucketPolicy'],
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketReplication',
+        permissions: ['s3:PutReplicationConfiguration'],
+        expectedResultOnAllowTest: 'InvalidRequest: Versioning must be \'Enabled\'',
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketVersioning',
+        permissions: ['s3:PutBucketVersioning'],
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutObjectLockConfiguration',
+        permissions: ['s3:PutBucketObjectLockConfiguration'],
+        expectedResultOnAllowTest: 'InvalidBucketState',
+    },
+    {
+        action: 'PutObject',
+        permissions: ['s3:PutObject'],
+    },
+    {
+        action: 'PutObjectLegalHold',
+        permissions: ['s3:PutObjectLegalHold'],
+        expectedResultOnAllowTest: 'MalformedXML',
+    },
+    {
+        action: 'PutObjectRetention',
+        permissions: ['s3:PutObjectRetention', 's3:BypassGovernanceRetention'],
+        needsSetup: true,
+    },
+    {
+        action: 'PutObjectTagging',
+        permissions: ['s3:PutObjectTagging'],
+    },
+    {
+        action: 'PutObjectAcl',
+        permissions: ['s3:PutObjectAcl'],
+    },
+    {
+        action: 'PutBucketCors',
+        permissions: ['s3:PutBucketCORS'],
+        expectedResultOnAllowTest: 'MissingRequestBodyError',
```

Comment:
same https://github.com/scality/Integration/blob/development/7.10/tests/bucket-policy/utils/s3APIConfigurations.js#L105

Reply:
Same for this one, no error, I can remove the check for consistency.
```
aws s3api put-bucket-cors --bucket zenko-ctstbucketfortestgzwcw --cors-configuration '{\"CORSRules\":[{\"AllowedMethods\":[\"GET\"],\"AllowedOrigins\":[\"*\"]}]}' --endpoint-url http://s3.zenko.local:80","error":null,"stdout":"","stderr":""}
```

Reply:
See https://github.com/scality/Zenko/pull/2023/commits/653404951d0bfc4b850d41a112541d0aa7e381d3